cibuild.sh

```diff
@@ -14,7 +20,36 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-cd src/Compilers/CSharp/csc
+echo Compiling using toolset compiler 
+docompile
+if [ $? -ne 0 ]; then
```

Comment:
You sure you want to be using bash here versus $(ANY_MODERN_SCRIPTING_LANGUAGE)?